frontends/p4/alias.h

```diff
@@ -123,6 +123,8 @@ class ReadsWrites : public Inspector {
     void postorder(const IR::Operation_Binary *expression) override {
         auto left = ::get(rw, expression->left);
         auto right = ::get(rw, expression->right);
+        CHECK_NULL(left);
```

Comment:
We use this a lot. Maybe we should implement some form of checked `::get`?

Reply:
Maybe... or just use `.at()` it does produce a standard exception instead of p4c one, but that is not a big problem in my opinion. Maybe we should have some sort of guidelines. I'd guess many new programmers don't even know there is `::get` until they find some in the code.